### ISSUE TITLE
Generating reviewer metadata

### ISSUE THREAD

**Issue by noamross** [ropensci/codemetar#45]
R-core had recently (with our urging) added an allowable MARC code of `rev` to persons in the `Authors@R` field of description, indicating individuals or organizations that reviewed the package.  It would be great if **codemetar** could recognize this and put information about these persons in the appropriate JSON-LD fields.  (My understanding is that this would be a `reviewedBy` field).  

Note that there's not a standard way to link to the reviews themselves, though the developing convention for rOpenSci is to include the URL in the comment field of the `person()` call, [e.g.](https://github.com/ropensci/tidyhydat/blob/master/DESCRIPTION#L4-L11),

```
Authors@R: c(person("Sam", "Albers", email = "sam.albers@gov.bc.ca", role = c("aut", "cre")),
    person("David", "Hutchinson", email = "david.hutchinson@canada.ca", role = "ctb"),
    person("Province of British Columbia", role = "cph"),
    person("Luke", "Winslow", role = "rev", 
            comment = "Reviewed for rOpenSci, see <https://github.com/ropensci/onboarding/issues/118>"),
    person("Laura", "DeCicco", role = "rev", 
            comment = "Reviewed for rOpenSci, see <https://github.com/ropensci/onboarding/issues/118>")
    )
```

**Comment by cboettig** [comment]
Thanks for the issue.  We'll resolve https://github.com/codemeta/codemeta/issues/177 and then it should be obvious how to proceed here.  

(Though not obvious if we could ever get this info into the DESCRIPTION structure)

**Comment by maelle** [comment]
Just adding a link (see what I did here) between these two issues https://github.com/ropensci/codemetar/issues/23

**Comment by maelle** [comment]
It'd be great to still have the info in the authors list of DESCRIPTION or at least in a field that appears on the CRAN page of the package [example](https://cran.r-project.org/web/packages/opencage/index.html)), because this increases visibility of rOpenSci onboarding reviews (although in the future maybe CRAN pages will digest stuff from codemeta.json).

**Comment by cboettig** [comment]
README links to the review via the ropensci badge.  I have mixed feeling about overloading the `comment` field for all the things.  There's a lot of additional data we might want to associate with people (e.g. affiliations are commonly part of author metadata, same for urls, but neither are part of a `person()` class.  It is also rather strange to think of a "review" as a property of a "person" in the way that a givenName, role, email or ORCID id are properties of that person).  

This could be better addressed by #23 I think.
